x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agents/agent_list_page/components/agent_list_table.tsx

```diff
@@ -268,7 +268,23 @@ export const AgentListTable: React.FC<Props> = (props: Props) => {
       }),
       width: '100px',
       render: (lastCheckin: string) =>
-        lastCheckin ? <FormattedRelative value={lastCheckin} /> : undefined,
+        lastCheckin ? (
+          <EuiToolTip
```

Comment:
Could we add a short text too, something like `Last seen online on February 10, 2025`? I think it would make it more clear for the user what's this date.